oci/client.py

```diff
@@ -11,6 +12,8 @@
 import threading
 import time
 
+import aiohttp
```

Comment:
ditto. preferably either make import optional (and raise if aiohttp is absent)? maybe an `client_async` module might also be an option

users might then use API like so:

```
import oci
import oci.client_async as ocm_client # OR: import ocm.client as oci_client
```

admittedly, it does not seem nice to have asymetry w.r.t. functions in `oci/__init__` :-/
-> maybe we should consider moving them into a submodule (e.g. `oci.replicate`) and deprecate the old ones (it should be feasible to make our userbase migrate so we can eventually cleanup) ?

such an approach would have the benefit that:
- we only claim one rootlevel-package
- modules w/o sync/async-relevance stay as they are
- others have companions, allowing users not needing / desiring async to avoid dependency by simply not importing async-modules
- we still have means to deduplicate / share common dataclasses / helper functions (e.g. async-companion-modules might import their non-async friends)